proto/cometbft/abci/v1beta4/types.proto

```diff
@@ -4,7 +4,7 @@ package cometbft.abci.v1beta4;
 import "cometbft/abci/v1beta1/types.proto";
 import "cometbft/abci/v1beta2/types.proto";
 import "cometbft/abci/v1beta3/types.proto";
-import "cometbft/crypto/v1beta1/proof.proto";
```

Comment:
Retroactive changes in `v1beta*` packages update references to other packages which were simply renamed from `v1beta1` to `v1`, because there has only ever been one version in `crypto`, `libs/bits`, etc. and it does not make sense to produce duplicate definitions for those domains.

The rest of the changes to `**/v1beta*/*` is whitespace formatting: I had a different formatter setup on file save prior to this, but this style is true to clang-format.

Reply:
I'm also not sure if we should preserve the latest beta packages like `cometbft.abci.v1beta4` here: they are functionally equivalent to the new `v1` definitions, the only difference is they back-reference definitions in earlier package versions whereas the `v1` set is self-contained.